query-engine/connectors/sql-query-connector/src/database/mysql.rs

```diff
@@ -39,31 +38,10 @@ fn get_connection_info(url: &str) -> connector::Result<ConnectionInfo> {
 #[async_trait]
 impl FromSource for Mysql {
     async fn from_source(
-        source: &psl::Datasource,
+        _: &psl::Datasource,
         url: &str,
         features: psl::PreviewFeatures,
     ) -> connector_interface::Result<Mysql> {
-        if source.provider == "@prisma/mysql" {
-            #[cfg(feature = "js-connectors")]
-            {
-                let driver = super::js::registered_driver();
-                let connection_info = get_connection_info(url)?;
-
-                return Ok(Mysql {
-                    pool: RuntimePool::Js(driver.unwrap().clone()),
-                    connection_info,
-                    features: features.to_owned(),
-                });
-            }
-
-            #[cfg(not(feature = "js-connectors"))]
-            {
-                return Err(ConnectorError::from_kind(ErrorKind::UnsupportedConnector(
-                    "The @prisma/mysql connector requires the `jsConnectors` preview feature to be enabled.".into(),
-                )));
-            }
-        }
-
```

Comment:
I just realised that removing this block means that we're now instantiating both the PlanetScale JS connector and the underlying Rust driver for MySQL. This is wrong and unwanted.